notebooks/1_LinearMethods.ipynb

```diff
@@ -1068,77 +1042,20 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "larger_npca = [10, 50, 100, 200]\n",
-    "\n",
-    "all_stats = [get_stats(y=Y_test, yp=Y_pca)]\n",
-    "headers = [r\"$n_{PCA}$\"+\"={}\".format(n_PC)]\n",
-    "\n",
-    "for n in larger_npca:\n",
-    "    pca2 = PCA(n_PC=n)\n",
-    "    pca2.fit(X_train)\n",
-    "\n",
-    "    T_train2 = pca2.transform(X_train)\n",
-    "    T_test2 = pca2.transform(X_test)\n",
-    "\n",
-    "    PTY2 = np.matmul(T_train2.T, T_train2) + best_regularization*np.eye(T_train2.shape[1])\n",
-    "    PTY2 = np.linalg.pinv(PTY2)\n",
-    "    PTY2 = np.matmul(PTY2, T_train2.T)\n",
-    "    PTY2 = np.matmul(PTY2, Y_train)\n",
```

Comment:
I just think it is better to keep it than to delete it. What is the reason for deleting it?

Reply:
So PCR gets better with larger numbers of PCs, but making it the focus here detracts from the overall focus of the notebook. Additionally, sklearn doesn't have a nice, easy PCR, because [they favor PLS and suggest just using LR on PCA for PCR](https://scikit-learn.org/dev/auto_examples/cross_decomposition/plot_pcr_vs_pls.html). 

I think there are far more reasons to delete it than "it is better to keep it".

Reply:
Okay, I agree.